### ISSUE TITLE
AutoCAD 2018 and Visual Studio 2017

### ISSUE THREAD

**Issue by sn0vv** [CADbloke/CADtest#15]
Hello, I'd like to use CADTest with AutoCAD 2018 and Visual Studio 2017.
But it's not successful. Please let me know some tweakage to get working.
 
1.	I downloaded CADtest-master.zip and unzipped it to C:\temp\CADtest-master
2.	I double clicked CADtest.sln to launch Visual Studio 2017.
3.	I deleted CADtest-VB because I'm a C# user.
4.	The first build finished successfully.
5.	I tried to start debug but Visual Studio couldn’t find “C:\Program Files\Autodesk\AutoCAD 2016\accoreconsole.exe,” so I changed the path to AutoCAD2018.
6.	I tried to run TestClass1(7) In Test Explorer, but that failed.
The output is following. * I translate the message from Japanese into English.

> [2019/10/18 11: 15: 59 Informational] ------ Test search has started ------
> [2019/10/18 11: 16:00 Informational] Test project CADtest-CS does not refer to. NET NuGet adapter. This project may not be able to detect or run tests.
> It is recommended that you refer to the NuGet test adapter in each test project in your solution.
> [2019/10/18 11: 16: 00 Informational] ========== Test search completed: 0 found (0: 00: 01. 2139375) ===== =====

I opened CADtest-CS.csproj but don't know what to be modified.

**Comment by CADbloke** [comment]
I haven’t used this library for quite a while. I recall having issues with the core console - only the 1st version worked properly like a console then Autodesk changed the behaviour. 

Did you try running it in AutoCAD and starting the tests with a command?

Also, the NUnit NuGet package is probably really out of date. 

I don’t have much time in the near future to look at this but I’ll try to get to it.

**Comment by sn0vv** [comment]
Thanks for your quick response.

I start running the tests by clicking Test Explorer on Visual Studio.
![cadTest](https://user-images.githubusercontent.com/10492222/67169456-57a57680-f3e6-11e9-9e57-5ae343ce3ff7.png)

**Comment by CADbloke** [comment]
That definitely won’t work. Have you read the wiki? There should be instructions in there on how to run the tests. I think. I might be thinking of the wrong project :/.  Spoiler alert: you run the from inside AutoCAD, not from Visual Studio.

**Comment by CADbloke** [comment]
Yeah nah wrong project. This readme is it. 
You need to use a command in AutoCAD to start the tests running. I’m on a phone tonight, I’ll try to get a look at it when I am next in front of a computer.

**Comment by CADbloke** [comment]
Here is the line in the code where it runs the tests: https://github.com/CADbloke/CADtest/blob/master/CADtest-CS/CADtestRunner.cs#L22

So you `NETLOAD` the dll into AutoCAD and then run the command `RunCADtests`

It does not run in Visual Studio, it only runs from inside AutoCAD.

**Comment by sn0vv** [comment]
I understand that I should have used 'RunCADtests' to run the tests.
Now CADTest seems to work.
Thanks a lot.

For someone's interest, I put my work records.
![debugSetting](https://user-images.githubusercontent.com/10492222/67375909-40b97c80-f5be-11e9-8a4b-8c6169936147.PNG)

AutoNetLoad2018.scr
```
netload "C:\temp\CADtest-master\CADtest-CS\bin\Debug\CADtest-CS.dll"RunCADtests
```

Hmm, Report-NUnit.xml is not generated...
![testRunner](https://user-images.githubusercontent.com/10492222/67376061-78282900-f5be-11e9-923b-129e8504b162.PNG)

**Comment by CADbloke** [comment]
**`System.IO.DirectoryNotFoundException`** - the `directoryReportUnit` does not exist (yet).

After this line: https://github.com/CADbloke/CADtest/blob/master/CADtest-CS/CADtestRunner.cs#L28

Add 
````cs
Directory.CreateDirectory(directoryReportUnit);
````

or pull this repo - I added it here.

**Comment by sn0vv** [comment]
I've added the line.
For someone's interest, I put my work records.

NUnitLite 3.5 works, but higher does not.
![nunitlite](https://user-images.githubusercontent.com/10492222/67556379-054db800-f74e-11e9-822e-4d40ba1b1b17.PNG)

I've built ReportUnit and put exe in directoryReportUnit.
https://github.com/reportunit/reportunit

Report-NUnit.html looks nice.
![reportUnit](https://user-images.githubusercontent.com/10492222/67556879-e8fe4b00-f74e-11e9-86bf-3845499290d6.PNG)
